src/style.css

```diff
@@ -0,0 +1,88 @@
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block {
+  flex-grow: 1;
+  min-width: 300px;
+  height: 300px;
+  color: white;
+  font-family: Arial, sans-serif;
+  font-size: 100px;
+  text-align: center;
+  line-height: 300px;
+}
+
+.block-1 {
+  background-color: rgb(255, 0, 0);
+}
+
+.block-2 {
+  background-color: rgb(204, 0, 0);
+}
+
+.block-3 {
+  background-color: rgb(153, 0, 0);
+}
+
+.block-4 {
+  background-color: rgb(102, 0, 0);
+}
+
+.block-5 {
+  background-color: rgb(51, 0, 0);
+}
+
+.block-6 {
+  background-color: black;
+}
+
+@media (min-width:600px) and (max-width: 900px) {
+  .block-1 {
+    order: 0;
+  }
+  .block-2 {
+    order: 1;
+  }
+  .block-3 {
+    order: 3;
+  }
+  .block-4 {
+    order: 2;
+  }
+  .block-5 {
+    order: 4;
+  }
+  .block-6 {
+    order: 5;
+  }
+}
+
+@media (min-width: 900px) {
+  .block {
+    width: 33.33%;
+  }
+  .block-1 {
+    order: 0;
+  }
+  .block-2 {
+    order: 1;
+  }
+  .block-3 {
+    order: 2;
+  }
+  .block-4 {
+    order: 5;
+  }
+  .block-5 {
+    order: 4;
+  }
+  .block-6 {
+    order: 3;
+  }
```

Comment:
do you really need to hardcode order for every block?

Reply:
not really)